config/config.go

```diff
@@ -23,8 +23,8 @@ type Config struct {
 	InsecureRegistries       []string                 `json:"insecureRegistries"`
 	CredentialsStore         string                   `json:"credsStore,omitempty"`
 	CredentialHelpers        map[string]string        `json:"credHelpers,omitempty"`
```

Comment:
Do we need `CredentialsStore`, `VerificationCertificates` and `CredentialHelpers` ?

Reply:
CredentialsStore and CredentialHelpers are using for login/logout command.
VerificationCertificates will be no longer used after TS/TP.